### PR TITLE
chore: Add questions

### PR DESCRIPTION
**Pull request by its0x08**
- Add question `C`
- Add questions `Agile`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1177,7 +1177,7 @@ Thanks goes to these wonderful people ([emoji key](https://allcontributors.org/d
...
     <td align="center"><a href="https://referal.ir/vahid.mafi"><img src="https://avatars.githubusercontent.com/u/70995302?v=4?s=100" width="100px;" alt=""/><br /><sub><b>Vahid Mafi</b></sub></a><br /><a href="#design-vahidmafi" title="Design">🎨</a> <a href="https://github.com/Ebazhanov/linkedin-skill-assessments-quizzes/commits?author=vahidmafi" title="Code">💻</a></td>
     <td align="center"><a href="https://github.com/AkshayAshokCode"><img src="https://avatars.githubusercontent.com/u/63944420?v=4?s=100" width="100px;" alt=""/><br /><sub><b>Akshay Ashok</b></sub></a><br /><a href="#design-AkshayAshokCode" title="Design">🎨</a> <a href="https://github.com/Ebazhanov/linkedin-skill-assessments-quizzes/commits?author=AkshayAshokCode" title="Code">💻</a></td>
-  </tr>
+     <td align="center"><a href="https://github.com/its0x08"><img src="https://avatars.githubusercontent.com/u/15280042?v=4&s=100" width="100px;" alt=""/><br /><sub><b>0x08</b></sub></a><br /><a href="#design-its0x08" title="Design">🎨</a> <a href="https://github.com/Ebazhanov/linkedin-skill-assessments-quizzes/commits?author=its0x08" title="Code">💻</a></td>
```
Comment (Ebazhanov) [inline README.md:1180]: no worries you will add automatically but there is a process, those line should generated by special command
please remove this lines

**Review by Ebazhanov — CHANGES_REQUESTED** [review]
Hello, @its0x08! Looks perfect and thanks for initiative to be one of the responsible for C programming language. 🙏  Going to approve after you remove line which usually generated automatically. Thanks

File: README.md (hunk 1 above)
Comment (its0x08) [reply README.md:1180]: Sure!
I thought it was made by hand 🤣

Comment (Ebazhanov) [reply README.md]: no it is by command fyi https://allcontributors.org/docs/en/bot/usage
waiting for update

**Review by Ebazhanov — APPROVED** [review]
@its0x08 thanks again 💪   we are good to go!!!

File: README.md (hunk 1 above)
Comment (its0x08) [reply README.md]: Done

**Comment by Ebazhanov** [comment]
@all-contributors please add @its0x08 for design, code

**Pull request merged**